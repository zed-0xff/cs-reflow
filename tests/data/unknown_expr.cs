void case1()
{
    int num16 = afd1f0100a9e48eb9a6c8975d80c2689;
    if (-67108864 + (int)((uint)num16 / 1024u) == (int)(2048 * (((uint)num16 % 1949u) ^ 0x70EF1C76)))
    {
        should_not_run();
    }
    else
    {
        should_run();
    }
}
