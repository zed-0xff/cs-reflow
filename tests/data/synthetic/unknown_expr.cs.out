void case1()
{
    should_run();
}

